.eslintrc.js

```diff
@@ -1,16 +1,24 @@
 module.exports = {
-  env: {
-    browser: true,
-    es2021: true,
-    node: true,
+  root: true,
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'next/core-web-vitals',
+    'prettier',
+  ],
+  rules: {
+    'no-console': 'error',
+    'no-shadow': 'error',
+    'react/jsx-sort-props': [
+      'error',
+      {
+        noSortAlphabetically: true,
+        shorthandLast: true,
+        callbacksLast: true,
+      },
+    ],
+    'react/no-array-index-key': 'warn',
```

Comment:
I prefer `error` for this rule, but sometimes there aren't other solution for `key`-prop. So I decided to use `warn` just in case to show developer that there is place for improvement.